src/resources/template.html

```diff
@@ -11,6 +11,7 @@
 			<meta name="theme-color" content="<%= htmlWebpackPlugin.options.manifest.theme_color %>">
 		<% } %>
 		<% for (var chunk of webpack.chunks) { %>
+		 <% if (chunk.names.length === 1 && chunk.names[0] === 'polyfills') continue; %>
 			<% for (var file of chunk.files) { %>
 				<% if (htmlWebpackPlugin.options.preload && file.match(/\.(js|css)$/)) { %>
```

Comment:
BTW `htmlWebpackPlugin.options.preload` comes from `argv.preload` which is a hidden switch that can't be used. Gonna check whole codebase for hidden switches sometime soon.